README.md

```diff
@@ -1,3 +1,5 @@
+![CI](https://github.com/slobentanzer/sctk/actions/workflows/ci-cd.yaml/badge.svg)
```

Comment:
this should be Teichlab instead, no?

Reply:
yes, once it is merged it should be changed to teichlab